src/seedu/addressbook/AddressBook.java

```diff
@@ -90,398 +43,90 @@
-     */
-
-    /**
-     * List of all persons in the address book.
-     */
     private static final ArrayList<String[]> ALL_PERSONS = new ArrayList<>();
-
-    /**
-     * Stores the most recent list of persons shown to the user as a result of a user command.
-     * This is a subset of the full list. Deleting persons in the pull list does not delete
-     * those persons from this list.
-     */
     private static ArrayList<String[]> latestPersonListingView = getAllPersonsInAddressBook(); // initial view is of all
-
-    /**
-     * The path to the file used for storing person data.
-     */
     private static String storageFilePath;
-
-    /*
-     * NOTE : =============================================================
-     * Notice how this method solves the whole problem at a very high level.
-     * We can understand the high-level logic of the program by reading this
-     * method alone.
-     * If the reader wants a deeper understanding of the solution, she can go
-     * to the next level of abstraction by reading the methods that are
-     * referenced by the high-level method below.
-     * ====================================================================
-     */
-
     public static void main(String[] args) {
-        showWelcomeMessage();
-        processProgramArgs(args);
-        loadDataFromStorage();
-        while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
-        }
-    }
-
-    /*
-     * NOTE : =============================================================
-     * The method header comment can be omitted if the method is trivial
-     * and the header comment is going to be almost identical to the method
-     * signature anyway.
-     * ====================================================================
-     */
-
-    private static void showWelcomeMessage() {
-        showToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
-    }
-
-    private static void showResultToUser(String result) {
-        showToUser(result, DIVIDER);
-    }
-
-    /*
-     * NOTE : =============================================================
-     * Parameter description can be omitted from the method header comment
-     * if the parameter name is self-explanatory.
-     * In the method below, '@param userInput' comment has been omitted.
-     * ====================================================================
-     */
-
-    /**
-     * Echoes the user input back to the user.
-     */
-    private static void echoUserCommand(String userCommand) {
+        showWelcomeMessage();if (args.length >= 2) {showToUser(MESSAGE_INVALID_PROGRAM_ARGS);exitProgram();}if (args.length == 1) {setupGivenFileForStorage(args[0]);}if(args.length == 0) {setupDefaultFileForStorage();}loadDataFromStorage();
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.